### ISSUE TITLE
Fullscreen resolution

### ISSUE THREAD

**Issue by HermannDppes** [UltraStar-Deluxe/USDX#504]
### Actual behaviour

When selecting “Fullscreen: On” and choosing a resolution different from / smaller than the native resolution of my display and restarting, only an upper left rectangle of the screen is visible but stretches over the entire display. The smaller the selected resolution the part that is visible and the larger the scaling of the elements of USDX.

### Expected behaviour

I selected this to see what it would do. I don't have a firm opinion on what it should do, but certainly *not this*.

### Steps to reproduce

1. Use a 1920 by 1080 pixel display.
2. Use the options or config.ini to
```
FullScreen=On
ResolutionFullscreen=1440x900
```
3. Start UltraStar.

### Details

This is on a nearly current master.

**Comment by basisbit** [comment]
Can not reproduce the problem / works on my machine. Please add details related to your system - operating system, hardware details, debug log file, ...

**Comment by HermannDppes** [comment]
Linux Mint 19.1.

[Error.log](https://github.com/UltraStar-Deluxe/USDX/files/4000775/Error.log)

**Comment by dgruss** [comment]
cannot reproduce on windows or linux

**Comment by barbeque-squared** [comment]
Yeah this is too vague to do anything about. _Especially_ the combination of fullscreen + non-native resolution, the actual physical display used also becomes part of the equation. For example most Dell monitors I've seen will allow you to choose between stretching to fit the physical pixels, scaling up but preserving aspect ratio, or just 1:1 it onto the center of the screen.

Somewhere in the chain these things are getting mixed up. I doubt it's USDX, but without details it's impossible to even suggest if this something in a graphics driver, Xorg, the physical display, the display cable itself, GPU risers, or anything else.

A first step would be to run literally _any_ other application in a fullscreen (**not borderless!**) mode to see what that produces. Until then I'm closing this as "Can't repro" (it's annoying that github shoves this into the same category as "Won't fix/not planned")
